app/rest_client/authClient.js

```diff
@@ -0,0 +1,11 @@
+import Client from './baseClient'
+
+const http = new Client('http://localhost:1111')
```

Comment:
You should this variable into an .env file

Reply:
> You should this variable into an .env file

yeah, I have tried before but my `.env` file not working bro

Reply:
I think you miss the documents!

Reply:
you can create a file called .env.local
and pass valuable like this:
`NEXT_PUBLIC_REACT_APP_API_URL=http://localhost:8000/`